doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -65,6 +65,7 @@ Working with nRF91 Series
   * :ref:`asset_tracker_v2`
```

Comment:
Line 57: Spacing issue. See the rendering here: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/releases_and_maturity/releases/release-notes-changelog.html#build-system